src/NHibernate/Properties/EmbeddedPropertyAccessor.cs

```diff
@@ -64,11 +79,25 @@ public override string ToString()
 		[Serializable]
 		public sealed class EmbeddedSetter : ISetter
 		{
-			private readonly System.Type _clazz;
+			[NonSerialized]
+			private System.Type _clazz;
+			private SerializableSystemType _serializableClazz;
```

Comment:
`EmbeddedSetter` usage of this field was not impacted by the type change, but for consistency with `EmbeddedGetter`, I have changed it too.